juan-aristizabal/tarea3/small-enought.js

```diff
@@ -0,0 +1,15 @@
+function smallEnough(a, limit) {
+  let numbers = a;
+  let limite = limit;
+  let bool = false;
+
+  for (let i = 0; i < numbers.length; i++) {
```

Comment:
@Juan-Aristizabal  , use modern approaches for array operations like forEach, map, reduce.